.travis.yml

```diff
@@ -0,0 +1,15 @@
+os: osx
+language: objective-c
+osx_image: xcode8.3
+node_js: 5.1.1
+
+before_install:
+  - cd server
+
+install:
+  - npm install
+
+script:
```

Comment:
(I have plans to try and add command-line building of the xcode project, too, but that will almost certainly be a future project, thanks to code signing…)